test-data/stdlib-samples/3.2/test/test_genericpath.py

```diff
@@ -23,7 +23,7 @@ def safe_rmdir(dirname: str) -> None:
 
 class GenericTest(unittest.TestCase):
     # The path module to be tested
-    pathmodule = genericpath # type: Any
+    pathmodule = genericpath
```

Comment:
Hmm, after further thought I think I understand your point: "type alias" is to "module alias" as "var of type Type" is to "var of type types.ModuleType." I'm still not sure that this provides the most intuitive or useful behavior, given that type aliases and module references are used in different ways.  I just tested and I can still pass a real module reference to a function that takes a parameter annotated as `types.ModuleType` -- is there anything I _can't_ do with a real module reference that I can do with an ordinary variable of `types.ModuleType`? Put differently, given your preferred behavior, what real-world use case is there to annotate a variable as `types.ModuleType`?

Anyway, changed my mind and pushing all changes as you suggest. Still would like to understand the reasoning, though.